### PR TITLE
Clean up: web service version

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

**Review by Peyman-N — APPROVED** [review]

File: schemas/products/webServiceVersion.schema.tpl.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add the content types of all possible input formats for this web service version.",
+      "_instruction": "Add all content types that can be used as input for this web service version.",
```
Comment (lzehl) [inline schemas/products/webServiceVersion.schema.tpl.json]: ... for... OR ... by... ?? @apdavison please advise

**Review by lzehl — COMMENTED** [review]
@apdavison I need your feedback here. I approve all other changes of this PR

File: schemas/products/webServiceVersion.schema.tpl.json (hunk 1 above)
Comment (apdavison) [reply schemas/products/webServiceVersion.schema.tpl.json]: by

File: schemas/products/webServiceVersion.schema.tpl.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
       "type": "array",
       "minItems": 1,
       "uniqueItems": true,
-      "_instruction": "Add the content types of all possible output formats for this web service version.",
+      "_instruction": "Add all content types that can be generated as output for this web service version.",
```
Comment (apdavison) [inline schemas/products/webServiceVersion.schema.tpl.json]: "output _by_"

**Review by apdavison — APPROVED** [review]

File: schemas/products/webServiceVersion.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/products/webServiceVersion.schema.tpl.json]: done

File: schemas/products/webServiceVersion.schema.tpl.json (hunk 2 above)
Comment (lzehl) [reply schemas/products/webServiceVersion.schema.tpl.json]: done

**Pull request merged**